tests/Doctrine/Tests/ORM/Tools/Console/Command/SchemaTool/CreateCommandTest.php

```diff
@@ -8,10 +8,15 @@
 
 class CreateCommandTest extends AbstractCommandTest
 {
+    /**
+     * @doesNotPerformAssertions
+     */
     public function testItPrintsTheSql(): void
     {
         $tester = $this->getCommandTester(CreateCommand::class);
         $tester->execute(['--dump-sql' => true]);
-        self::assertStringContainsString('CREATE TABLE keyboard', $tester->getDisplay());
+
+        $statements = $tester->getDisplay();
+        self::$sharedConn->executeStatement($statements);
```

Comment:
Will this really work with several statements? Or should you use a foreach?

Reply:
it does not throw any exception with multiple statements. but since it's a bunch of raw queries `exec()` might be more appropriate?

Reply:
Didn't know this `method()`, but given its phpdoc comment, I'd advise against using it, it seems bound to be removed. Also, take a look at the implementation: you gain nothing by using it. To know if it works with several statement, try running a second statement that is syntactically valid but will break, like deleting a table that does not exist.

Reply:
aha. `exec()` just delegates to `executeStatement()` 👀 

https://github.com/doctrine/dbal/blob/39a05f63ffb95baf13716a6c11e268a720bf2552/src/Connection.php#L1846-L1854

so we're good i guess.

Reply:
> To know if it works with several statement, try running a second statement that is syntactically valid but will break, like deleting a table that does not exist.

works.